config/environments/development.rb

```diff
@@ -36,7 +35,6 @@
   config.assets.debug = true
 
   config.action_mailer.default_url_options = { :host => 'my-library-nyc-app-development.us-east-1.elasticbeanstalk.com' }
-  config.action_mailer.perform_deliveries = true
 
```

Comment:
Is config.action_mailer.perform_deliveries no longer a setting?

Reply:
No. It's being set config/initializers/setup_mail.rb